doc/command_ref.rst

```diff
@@ -192,6 +192,10 @@ Options
     ``download``, ``modulesync`` or ``reposync`` commands (dnf-plugins-core) or with the ``system-upgrade`` command
     (dnf-plugins-extras).
 
+    When used with the ``--downloadonly`` option, packages are removed after the next successful transaction
+    as the directory is considered as a part of the DNF cache. To persist the packages, use the ``download``
+    command instead.
+
 .. _downloadonly-label:
 
 ``--downloadonly``
```

Comment:
Maybe add a note also to the `--downloadonly` option? If I understand correctly, when `--downloadonly` is used, the packages are removed after the next successful transaction regardless of the `--destdir` option, so one might use only `--downloadonly` without `--destdir` (therefore not reading the documentation for `--destdir`) and be surprised as well.

Reply:
You're right. I am thinking now about moving this note just to `--downloadonly` option to prevent bloating the docs. Because user would probably only care about this when explicitely downloading the package, right?

Reply:
Yeah, you are right, I think having it in the `--downloadonly` option would be enough. But I would then explicitly mention that the packages will be deleted even when `--destdir` is provided.